environment.yml

```
@@ -26,4 +26,5 @@ dependencies:
   - tqdm
   - requests
   - asf_search
-
+  - pip:
+      - hyp3-sdk
```

```suggestion
  - hyp3_sdk
```
HyP3 SDK is also available via conda